tests/add_i256/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+#![no_std]
+use soroban_sdk::{contractimpl, I256};
+
+pub struct Contract;
+
+#[contractimpl]
+impl Contract {
+    pub fn i256(i256: I256) -> I256 {
+        i256
+    }
+}
+
+#[cfg(test)]
+mod test {
+    use soroban_sdk::{Env, I256};
+
+    use crate::{Contract, ContractClient};
+
+    #[test]
+    fn test_id() {
+        let e = Env::default();
+        let contract_id = e.register_contract(None, Contract);
+        let client = ContractClient::new(&e, &contract_id);
+
+        let y = I256::from(4u128.pow(600));
```

Comment:
Looks like the github action is failing due to a type mismatch here, but besides that, wouldn't `4u128.pow(600)` overflow i256?